### DISCUSSION TITLE
Welcome to 30DayMapChallenge2024 Discussions!

### DISCUSSION THREAD

**Discussion by cforgaci** [Rbanism/30DayMapChallenge2024#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ClementineCttn** [comment]
Hi All,
I'm Clémentine, I co-founded Rbanism (and am so glad the challenge got so many people interested!).
I have a question for mappers: who would be interested in an online meeting somewhere halfway into the challenge? Additionally, 
If you know someone interested in raster (6 Nov), please invite them into the challenge !
If you know someone interested in the Arctic (11 Nov), please invite them into the challenge !
If you know someone interested in the circular shapes (27 Nov), please invite them into the challenge !
If you know someone interested in the Overture Maps Foundation (29 Nov), please invite them into the challenge !
If you have ideas for the final map (30Nov), please let us know!
Have a nice November full of maps
Clémentine
